lib/main_common.pm

```diff
@@ -1926,8 +1926,6 @@ sub load_x11_installation {
     load_reboot_tests();
     loadtest "x11/x11_setup";
     loadtest 'qa_automation/patch_and_reboot' if is_updates_tests;
-    # temporary adding test modules which applies hacks for missing parts in sle15
-    loadtest "console/sle15_workarounds" if is_sle('15+');
```

Comment:
I'm not sure if we should just exclude it, but rather replace with `system_prepare`.